indexing-service/src/main/java/org/apache/druid/indexing/common/task/HadoopIndexTask.java

```diff
@@ -412,6 +430,57 @@ private TaskStatus runInternal(TaskToolbox toolbox) throws Exception
     }
   }
 
+  @Override
+  public void stopGracefully()
+  {
+    final ClassLoader oldLoader = Thread.currentThread().getContextClassLoader();
+    ObjectMapper objectMapper = new ObjectMapper();
+    File hadoopJobIdFile = new File(getHadoopJobIdFileName());
+    String jobId = null;
+
+    try {
+      if (hadoopJobIdFile.exists()) {
+        jobId = objectMapper.readValue(hadoopJobIdFile, String.class);
+      }
+    }
+    catch (Exception e) {
+      log.info("Exeption while reading json file from path: " + hadoopJobIdFile);
+      log.error(Throwables.getStackTraceAsString(e));
```

Comment:
You don't need to print two logs. Please replace them with `log.warn(e, "Exeption while reading Hadoop Job ID from: %s", hadoopJobIdFile);`.